package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@lemonpeach/sequelize-to-json-schema",
+  "version": "0.0.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "clean": "shx rm -rf lib",
+    "build": "npm run clean && babel src -d lib",
+    "lint": "eslint src",
+    "test": "jest src"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Lemonpeach/sequelize-to-json-schema.git"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
I believe this is the wrong license.  It should be MPL2. I forget the version code.

Reply:
MPL2.0 done